src/GovUk.Education.ExploreEducationStatistics.Publisher/Services/ReleaseService.cs

```diff
@@ -73,9 +73,11 @@ public async Task<Release> Get(Guid id)
         public async Task<IEnumerable<Release>> List(IEnumerable<Guid> ids)
         {
             return await _contentDbContext.Releases
+                .AsQueryable()
```

Comment:
I'm not sure that I follow any of these changes to Publisher.Services.ReleaseService. Are they improving performance? Thanks.

Reply:
We need to add these in `Publisher` as it depends on `Content.Services`. As I've added the `System.Linq.Async` package, this causes ambiguity in the compiler over which extension methods to use as `DbSet` implements both `IAsyncEnumerable` and `IQueryable`.

Consequently, we need to add `AsQueryable` to tell the compiler we're working with `IQueryable` specifically, and not an `IAsyncEnumerable`.

Reply:
Thanks, I understand now! 🙂